flocker/node/test/test_deploy.py

```diff
@@ -961,7 +961,7 @@ def test_discover_remotely_owned_volumes_ignored(self):
                     activation_state=u'active')
         units = {unit.name: unit}
 
-        volume = Volume(uuid=unicode(uuid4()),
+        volume = Volume(node_id=unicode(uuid4()),
                         name=_to_volume_name(u"site-example.com"),
```

Comment:
Again hopefully fixed in FLOC-697,or maybe followup task depending how big it is.